ods_ci/mysql_link

```diff
@@ -0,0 +1 @@
+tests/Resources/CLI/ModelRegistry/samples/mysql
```

Comment:
Is this file needed?

Reply:
All of the files for model-registry deployment are needed to deploy the service until it is fully available in ODH, by default model-registry is :removed.   
This will likely change once it gets into ODH and I can refactor the test at that point. For now though they are needed.

Reply:
There is also the consideration that it has to run on a disconnected cluster. As I understand, it is not possible to use an external reference from within a disconnected cluster to apply model-registry.

Reply:
I understand you need most files to set up Model Registry until it is fully integrated, but i was wondering if this `mysql_link` is actually needed, it seems to be a symlink to another directory?

Reply:
>As I understand, it is not possible to use an external reference from within a disconnected cluster to apply model-registry

In general yes, you're correct, in disconnected it's not possible to fetch resources remotely from the cluster itself.
But if you need the ods-ci tests to fetch resources and then apply them from local to the cluster, that is possible in disconnected.
However this seems to be a temporary problem that will go away once ODH/RHOAI can actually set everything up for Model Registry?

Reply:
I think I will leave it the way it is for now as it is temporary. I will update appropriately once the changes are actioned in model-registry-operator.